dask/tests/test_ci.py

```diff
@@ -35,5 +35,5 @@ def test_upstream_packages_installed():
         # "zict",
     ]
     for package in packages:
-        v = Version(importlib.metadata.version(package))
+        v = Version(importlib_metadata.version(package))
```

Comment:
Is `importlib` version parsing also buggy? If so, using `importlib_metadata` here sounds good, but if not I'd prefer to stick with `importlib.metadata` from the stdlib where possible

Reply:
I think it's cleaner to always use the backport throughout the whole project rather than have different versions of importlib[._]metadata used in different places

Reply:
especially now that this PR removes entry_points from `dask.compatibility` I don't want to have some modules with `import importlib_metadata` and others with `import importlib.metadata`

Reply:
I disagree this is cleaner. That said, the only place this comes up is in a single test that's `xfail`ed -- so not a big deal either way. Let's just go with what you have here